src/components/MessageList/MessageList.tsx

```diff
@@ -359,7 +368,9 @@ export const MessageList = <
     return null;
   };
 
-  const handleScroll: ScrollViewProps['onScroll'] = (event) => {
+  const handleScroll: ScrollViewProps['onScroll'] = (
+    event: NativeSyntheticEvent<NativeScrollEvent>,
+  ) => {
```

Comment:
this type should already be on the event object
```suggestion
  const handleScroll: ScrollViewProps['onScroll'] = (event) => {
```